novawallet/Modules/Vote/Governance/ReferendumVoteSetup/ReferendumVoteSetupViewFactory.swift

```diff
@@ -108,7 +108,7 @@ struct ReferendumVoteSetupViewFactory {
     // swiftlint:disable function_body_length
     private static func createInteractor(
         for state: GovernanceSharedState,
-        referendum: ReferendumIdLocal,
+        referendum _: ReferendumIdLocal,
```

Comment:
Looks like we forgot to provide referendum id to the module to be able to detect which referendum changed

Reply:
It's redundant as we now pass referendumId to presenter for this reason.